expo/scheduler/views.py

```diff
@@ -980,12 +977,16 @@ def calendar_view(request=None,
     # I want to rewrite this to get the first day of the event from the
     # DB or conf, and then calculate the date based on the next day after
     # the start of the event.  -  HH
-    if day is not None:
-        cal_times = day_to_cal_time(day,
-                                    week=datetime(2016,
-                                                  02,
-                                                  4,
-                                                  tzinfo=pytz.timezone('UTC')))
+    conf_slug = request.GET.get('conf', None)
```

Comment:
Small nit - but could we use either 'conference' or 'conf_slug' for this name for the get.  We use either of those things for the conference slug marker and consistency is nice.